src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -127,10 +127,7 @@ public AddressBook load() throws StorageOperationException {
 
         // create empty file if not found
         } catch (FileNotFoundException fnfe) {
-            final AddressBook empty = new AddressBook();
-            save(empty);
-            System.out.println("File was not found in file path but has been successfully recreated");
-            return empty;
+        	throw new StorageOperationException("File has been deleted");
```

Comment:
It is more meaningful to do the exception handling of the deleted file in save()